.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>StarRod</name>
```

Comment:
necessary?

Reply:
Looks like something the extension just decided to do on its own